### ISSUE TITLE
Julia 1.8.3 is not recognized as latest version

### ISSUE THREAD

**Issue by roflmaostc** [JuliaLang/juliaup#436]
Hi,

tested on two different Linux machines:
```bash
╰─➤  juliaup --version                                                                                                                  [18-11-22 | 10:44:56]
Juliaup 1.7.24

╰─➤  juliaup self update                                                                                                                [18-11-22 | 10:46:55]
Checking for self-updates
Juliaup unchanged on channel 'release' - 1.7.24

╰─➤  juliaup add 1.8.3                                                                                                                  [18-11-22 | 10:47:03]
Error: '1.8.3' is not a valid Julia version or channel name.
```

Is that a known issue or what is causing this?

Best,
Felix

**Comment by roflmaostc** [comment]
I see, will be fixed by #417

**Comment by uncomfyhalomacro** [comment]
> yes I think this manual addition to `versions.json` delay things. It requires a person to do it and then requires a person(from team) to approve merge it :(

i find it not good as well
